### ISSUE TITLE
CORS issues + 404/501 errors

### ISSUE THREAD

**Issue by KevinKelbie** [opentimestamps/javascript-opentimestamps#61]
Hi, sorry if I'm making obvious mistakes but its my first time using this library. I written the following code:

```
timestamp("d04b98f48e8f8bcc15c6ae5ac050801cd6dcfd428fb5f9e65c4e16e7807340fa")

function timestamp(hash) {
        const hash_ = Buffer.from(hash, 'hex');
        const detached = OpenTimestamps.DetachedTimestampFile.fromHash(new OpenTimestamps.Ops.OpSHA256(), hash_);
        OpenTimestamps.stamp(detached).then(() => {
            let fileOts = detached.serializeToBytes();
        });
    }
```

But it doesn't seem to be timestamping the hash instead its thowing CORS, 404, and 501 errors.

<img width="547" alt="Screenshot 2019-05-19 at 10 03 29" src="https://user-images.githubusercontent.com/19197564/57980029-69d94e00-7a1d-11e9-91f7-ca02fc023b39.png">
<img width="1202" alt="Screenshot 2019-05-19 at 10 03 17" src="https://user-images.githubusercontent.com/19197564/57980073-1b787f00-7a1e-11e9-8aed-1bed1075cd1c.png">

Any help is appretiated.

**Comment by lvaccaro** [comment]
Hi Kevin, f628d0d60a32a9e694cca74b2df2d3c2d107e20f should fix this issue (inside the v0.4.5 release), in order to prevent browsers from triggering CORS pre-flight requests.

**Comment by KevinKelbie** [comment]
I'm going to close this because I no longer have the code I was using in this issue. If anyone has the same issue in the future and wants to reopen it feel free.
